### ISSUE TITLE
kubeScheduler.usePolicyConfigMap doesn't work duo to a missing flag on kube-scheduler

### ISSUE THREAD

**Issue by waldman** [kubernetes/kops#4725]
1. What `kops` version are you running? The command `kops version`, will display
 this information.
1.9.0 alpha

2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.
1.9.3

3. What cloud provider are you using?
AWS

4. What commands did you run?  What is the simplest way to reproduce this issue?
I put up a cluster and set the useConfigMap Scheduler option:
```
  kubeScheduler:
    usePolicyConfigMap: true
```
With this option set the `kube-scheduler` breaks duo to the missing ` --policy-configmap-namespace`. This is a bug and here is the report for it on K8s upstream project: https://github.com/kubernetes/kubernetes/issues/61368

5. What happened after the commands executed?
The cluster get created but `kube-scheduler` doesn't get up with the following error: 
```
F0319 19:21:47.831522       1 helpers.go:119] error: couldn't get policy config map /scheduler-policy: an empty namespace may not be set when a resource name is provided
```

6. What did you expect to happen?
The scheduler to work using the ConfigMap stored at the kube-system namespace.

7. Please provide your cluster manifest. Execute
  `kops get --name my.example.com -oyaml` to display your cluster manifest.
  You may want to remove your cluster name and other sensitive information.

8. Please run the commands with most verbose logging by adding the `-v 10` flag.
  Paste the logs into this report, or in a gist and provide the gist link here.

9. Anything else do we need to know?
I'm appending a PR with the change shortly

**Comment by justinsb** [comment]
Believed fixed by #4726 - please reopen if not!

**Comment by justinsb** [comment]
(& thanks for fixing @waldman :-) )
